### PR TITLE
[FEAT] RBF - check for unconfirmed inputs

### PR DESCRIPTION
**Pull request by gabrielbazan7**

### CODE REVIEW

**Comment by micahriggan** [comment]
I would prefer we store the sequence number on the coin rather than a flag. So instead of isRBF could you add sequenceNumber, and populate that from the inputs as coins are being minted? 

In the API transform you can populate the isRBF flag if you'd like by doing the same comparison logic

File: packages/bitcore-node/src/models/coin.ts [hunk 1]
```diff
@@ -199,7 +200,8 @@ export class CoinModel extends BaseModel<ICoin> {
       address: valueOrDefault(coin.address, ''),
       script: valueOrDefault(coin.script, Buffer.alloc(0)).toString('hex'),
       value: valueOrDefault(coin.value, -1),
-      confirmations: valueOrDefault(coin.confirmations, -1)
+      confirmations: valueOrDefault(coin.confirmations, -1),
+      sequenceNumber: valueOrDefault(coin.sequenceNumber, undefined)
```
Comment (matiu) [inline packages/bitcore-node/src/models/coin.ts]: what's the point of this? (default = undefined will only be use if sequenceNumber=0?)

**Comment by gabrielbazan7** [comment]
> I would prefer we store the sequence number on the coin rather than a flag on transactions. So instead of isRBF could you add sequenceNumber, and populate that from the inputs as coins are being minted?
> 
> In the API transform you can populate the isRBF flag if you'd like by doing the same comparison logic.
> 
> This MR is probably the simplest way to accomplish the goal though. Just seems like we should have the sequence number in the db...
> 
> what do you think?

Done. Thanks for the comment

**Comment by matiu** [comment]
need rebase

**Pull request merged**